src/utils/functions.tsx

```diff
@@ -0,0 +1,91 @@
+import { Person } from '../types';
+import { PersonSex } from '../types/Filters';
+import { SortType } from '../types/SortType';
+
+function checkQuery(string: string, query: string): boolean {
+  const preparedString = string?.toLowerCase().trim();
+  const preparedQuery = query.toLowerCase().trim();
+
+  return preparedString.includes(preparedQuery);
+}
+
+export function getPreparedPeople(
```

Comment:
I think it would be good to cache, and so that there are no extra checks and renders, I mean to do through useMemo,
But i'm not sure